querier.hh

```diff
@@ -169,6 +185,17 @@ public:
                     cstats.clustering_rows.live,
                     cstats.clustering_rows.dead,
                     cstats.range_tombstones);
+            auto dead = cstats.static_rows.dead + cstats.clustering_rows.dead + cstats.range_tombstones;
+            if (_qr_config.tombstone_warn_threshold > 0 && dead >= _qr_config.tombstone_warn_threshold) {
+                auto live = cstats.static_rows.live + cstats.clustering_rows.live;
+                if (_range->is_singular()) {
+                    qrlogger.warn("Read {} live rows and {} tombstones for {}.{} pk \"{}\" {} (see tombstone_warn_threshold)",
+                                  live, dead, _schema->ks_name(), _schema->cf_name(), _range->start()->value().key()->with_schema(*_schema), (*_range));
+                } else {
+                    qrlogger.warn("Read {} live rows and {} tombstones for {}.{} pk <partition-range-scan> {} (see tombstone_warn_threshold)",
```

Comment:
"pk" can be removed from the message, so it will be:
```
"Read {} live rows and {} tombstones for {}.{} <partition-range-scan> {} (see tombstone_warn_threshold)",
```

Reply:
Ok. Also remove "pk" from the `pk \"{}\"` line?

Reply:
No, there it is fine as we do have a partition key. But you can replace it with "partition key" as users might not know ehat "pk" is.